README.md

```diff
@@ -15,8 +15,6 @@ module "MLOps" {
   data_location_s3        = "your_bucket/your_data.csv"
   model_target_variable   = "y"
   model_name              = "your-ml-model"
-  account_id              = var.account_id
```

Comment:
We shouldn't need to pass in the `account_id` and `region` as variables. These should be accessible (if needed) from data blocks (see new `data.tf` file)